### ISSUE TITLE
Color value is not displayed in native Cover Block custom color picker

### ISSUE THREAD

**Issue by derekblank** [WordPress/gutenberg#51283]
## What problem does this address?
In the native mobile Cover Block cutom color picker, the color value is not displayed as a text or numerical value when selecting a custom color (Hex, RGB, HSL). For color contrast accessibility reasons, it is helpful to have the value of the color displayed in a text or number format other than the color itself. Adding this value to the native color picker would also match the web behavior, which displays the Hex, RGB, and HSL value of the color selected. We could start with displaying the Hex values first, and the RBG and HSL values could be iterated on later.

## What is your proposed solution?
Within the mobile Cover Block custom color picker, display the color value similar to the web behavior and other mobile custom color pickers:

Web Editor (displays hex value) | Mobile Editor (is missing hex value)
-|-
<img width="375" alt="Screenshot 2023-06-07 at 3 27 57 pm" src="https://github.com/WordPress/gutenberg/assets/643285/c5c63086-7db4-41b2-b54f-ee8f0a4ec1f7"> | ![IMG_3023](https://github.com/WordPress/gutenberg/assets/643285/688fc011-588e-44b1-8bc6-5ff2489b082c)

**Comment by derekblank** [comment]
Completed in https://github.com/WordPress/gutenberg/pull/51414.
